eclipse/build.gradle.kts

```diff
@@ -58,4 +63,131 @@ tasks {
     artifacts {
         add("testing", testJar)
     }
+
+
+    /* Eclipse release tasks
+     *
+     * The following tasks provide the functionality of creating
+     * an eclipse update-site (via "updateSite") or dropin (via "dropin").
+     * The creation of the update-site uses as recommended the eclipse's
+     * P2Directory. You can find a how-to here:
+     * http://maksim.sorokin.dk/it/2010/11/26/creating-a-p2-repository-from-features-and-plugins/
+     *
+     * Instead of calling the P2Director via CLI we use the goomph
+     * plugin's classes that provide an abstraction layer of the P2Director.
+     */
+
+    val updateSiteDirPath = "build/update-site"
+
+    /* Step 0 of update-site creation
+     *
+     * Creates the structure:
+     * update-site/
+     *  |- features/
+     *    |- feature.xml
+     *  |- plugins/
+     *    |- saros.core.jar
+     *    |- saros.eclipse.jar
+     *  |- category.xml
+     */
+    val updateSitePreparation by registering(Copy::class) {
+        dependsOn(":saros.core:jar", ":saros.eclipse:jar")
+
+        from("feature/feature.xml") {
+            into("features")
+        }
+        into("plugins") {
+            from(project.tasks.findByName("jar"))
+            from(project(":saros.core").tasks.findByName("jar"))
+        }
+        from("update/site.xml")
+        into(updateSiteDirPath)
```

Comment:
Unify usage of `into...from..` logic? This uses 3 layouts. from->into with braces, into->from with braces, and from->into without braces.

Reply:
I think I could change the first closure into a `into{from}` (removing one layout).

However, the remaining two layouts are not interchangeable:
* The "plain" layout at the bottom of the task (l.103-104) defines the root target dir (`into`) and what should be copied into the root dir (`from`)
* The previous from/into-definitions define target dirs relative to the root dir.